README.md

```diff
@@ -36,6 +36,7 @@ that are implemented.
 | `Info()`         | x      | x     | x       |
 | `Memory()`       | x      | x     | x       |
 | `CPUTimer`       | x      | x     | x       |
+| `VMStat`         |        | x     |         |
```

Comment:
Just curious, why `Info` and `Memory` are here as function calls? Could we remove the `()`?

Reply:
I'm honestly not sure. Some of them have `()` and some don't.

Reply:
I think some are functions defined on the `Host` and others are interfaces implemented by the `Host`. Probably changing them all to be function names regardless of how they are defined is better.